### ISSUE TITLE
toHTML is not working anymore for a specific bold string

### ISSUE THREAD

**Issue by mos379** [xoofx/markdig#614]
```
**basics | 8:00**
```
does not get translated anymore and is simply put in a paragraph.
used to work before, not sure which version though

**Comment by xoofx** [comment]
Might be related to a recent fix 983187eace6ba02ee16d1443c387267ad6e78f58
Seems to happen only with advanced extensions, maybe with pipe tables.
Will need investigation.

**Comment by mos379** [comment]
the following is being used for initialization
MarkdownPipeline pipeline = new MarkdownPipelineBuilder().UseCustomContainers().UseGenericAttributes().UseAdvancedExtensions().Build();

and the problematic text is directly followed by a list ordered or unordered does not make a difference
```
**cycling | 8:00**
**Set-up:**
1) Hips higher than knees but lower than shoulders.
2) Chest up.
3) Opposite hand off to the side.
```

```
<p>**cycling | 8:00**
**Set-up:**</p>
<ol>
<li>Hips higher than knees but lower than shoulders.</li>
<li>Chest up.</li>
<li>Opposite hand off to the side.</li>
</ol>
```

**Comment by xoofx** [comment]
This should be fixed by commit 8beb096 and available in next version
